src/main/java/com/github/dgroup/dockertest/Logo.java

```diff
@@ -71,21 +69,23 @@ public Logo(final Scalar<String> version) {
      * @checkstyle OperatorWrapCheck (50 lines)
      * @checkstyle RegexpSinglelineCheck (50 lines)
      * @checkstyle StringLiteralsConcatenationCheck (50 lines)
+     * @checkstyle LocalFinalVariableNameCheck (50 lines)
      */
     @SuppressWarnings("PMD.AddEmptyString")
     public String asString() {
-        return new TextOf(
-            "\nDocker testing tool (%s)\n%s",
+        final String ls = System.lineSeparator();
+        return new TextOf(System.lineSeparator() +
+            "Docker image testing (%s)%s" + ls,
```

Comment:
@bodax, 4 things here:
1. let's rename the `ls` to `eol` (end of line);
2. line 78, use `eol` before `Docker testing tool` instead of using System.lineSeparator() again;
3. line 78 you are changed the logo due to `(%s)%s" + ls` so it should be like `(%s)" + eol + "%s"`;
4. line 80 should not be ending with `ls`, check the original implementation.